discovery-infra/test_infra/tools/static_network.py

```diff
@@ -0,0 +1,90 @@
+import json
+import os
+import random
+import yaml
+from ipaddress import ip_network, ip_address
+
+from test_infra import consts, utils
+
+
+def generate_macs(count):
+    return ["02:00:00:%02x:%02x:%02x" % (random.randint(0, 255), random.randint(0, 255), random.randint(0, 255))
+            for _ in range(count)]
+
+
+def generate_static_network_data_from_tf(tf_folder):
+    tfvars_json_file = os.path.join(tf_folder, consts.TFVARS_JSON_NAME)
+    with open(tfvars_json_file) as _file:
+        tfvars = json.load(_file)
+
+    network_config = []
+
+    for count in range(tfvars['master_count']):
```

Comment:
should there be another loop for worker_count ?

Reply:
fixed